src/compiler.js

```diff
@@ -480,6 +495,64 @@ class Compiler {
               )
         })
 
+        if (subgraph.has('mutations')) {
+          // Copy mutation files and update their paths
+          subgraph = subgraph.update('mutations', mutations =>
+            // mutations/schema.graphql + mutations/resolvers/index.js
```

Comment:
Why not just `mutations/resolvers.js` or `mutations/index.js` instead of adding the extra `resolvers` directory?

Reply:
I was thinking that the extra directory would be nice if we ended up having additional files alongside the index.js. The main example I was thinking of was type definitions, so index.d.ts. I suppose these both could be included at the root directory and it would still be fine, as name conflicts shouldn't be an issue. I'll move it up to the root directory.